libsolidity/parsing/Token.cpp

```diff
@@ -101,17 +108,21 @@ char const Token::m_tokenType[] =
 {
 	TOKEN_LIST(KT, KK)
 };
-unsigned Token::extractM(string const& _literal)
+unsigned Token::extractUnsigned(string::const_iterator const& _begin, string::const_iterator const& _end)
```

Comment:
I've been trying to figure out how to make this work with iterators but nothing is clicking so far.